src/main/java/com/meilisearch/sdk/Documents.java

```diff
@@ -15,16 +15,36 @@ protected Documents(Config config) {
 		meilisearchHttpRequest = new MeiliSearchHttpRequest(config);
 	}
 
+	/**
+	 * Retrieves the document at the specified uid with the specified identifier
+	 * @param uid Partial path to the requested document
+	 * @param identifier ID of the document
+	 * @return Requested document
```

Comment:
```suggestion
	 * @return A String containint the requested document
```
Good, but could be interesting to mention the type! 👍